__tests__/components/Comments/ContainerComment.test.js

```diff
@@ -218,7 +222,8 @@ describe('ContainerComment should run correctly', () => {
     const findCommentAndUpdate = jest.fn();
 
     const contextValue = {
-      feeds: [feedsState, () => jest.fn()]
+      feeds: [feedsState, () => jest.fn()],
+      profile: [{}, () => jest.fn()]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the `profile` state is added to the context value. As mentioned before, ensure that this state is properly initialized and updated in the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->